src/decorators/au.ts

```diff
@@ -60,6 +63,15 @@ const decorator: IDecorator = {
         }
       }
 
+      if (digit === "x") {
+        decoratedNumber.parts.push(extensionDecorativePart(" ext. "));

```

Comment:
I think we need baked in support for localization here. Obviously on the LoopUp side, but also the purity of the library as a whole shouldn't have us depending on hardcoded strings